static/app/components/events/featureFlags/featureFlagOnboardingLayout.tsx

```diff
@@ -95,28 +93,32 @@ export function FeatureFlagOnboardingLayout({
   return (
     <AuthTokenGeneratorProvider projectSlug={projectSlug}>
       <Wrapper>
-        {!skipConfig ? null : (
+        {!skipEvalTracking ? null : (
```

Comment:
can this be like `{skipEvalTracking ? <Alert>...</Alert> : null}` instead of `!skipEvalTracking`

phrase the condition in the positive.